analyses/molecular-subtyping-ATRT/01-ATRT-molecular-subtyping-data-prep.Rmd

```diff
@@ -329,44 +355,82 @@ tmb_df %>%
 # Join ATRT expression, focal CN data and transposed ssGSEA data with tumor
 # mutation burden data
 atrt_expression_cn_tmb_df <- atrt_expression_cn_df %>%
-  dplyr::left_join(tmb_df, by = "sample_id")
+  dplyr::left_join(dplyr::select(tmb_df,
+                                 -Tumor_Sample_Barcode,
+                                 -Kids_First_Participant_ID), 
+                   by = "sample_id")
 
-## TODO: Add a column to this data.frame denoting `chr22q` loss using the SV
-# data.
+# Remove data we no longer need
+rm(tmb_df, atrt_expression_cn_df)
+```
+
+### GISTIC data
+
+```{r}
+gistic_df <- gistic_df %>%
+  dplyr::select(-Kids_First_Biospecimen_ID) %>%
+  dplyr::group_by(sample_id) %>%
+  dplyr::mutate(chr_22q_loss = dplyr::case_when(`22q` == -1 ~ "Yes",
```

Comment:
@jharenza we only have `-1` and `0` values in `22q` for these samples: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/7f012635616c171531b627606f6607874db04d30/analyses/molecular-subtyping-ATRT/atrt-subset/atrt_gistic_broad_values.tsv

Unless GISTIC itself incorporates the tumor ploidy information from `pbta-histologies.tsv`, we are not using any info about ploidy here. I think given that a `-1` would be a loss even when ploidy is 2, this might be fine but wanted to make you aware.

Reply:
yes, GISTIC only uses -2, -1, 0, 1, 2 to my knowledge